ext/mbstring/tests/big5_encoding.phpt

```diff
@@ -33,7 +33,17 @@ testAllValidChars($fromUnicode, 'UTF-16BE', 'BIG5', false);
 findInvalidChars($fromUnicode, $invalid, $unused, array_fill_keys(range(0,0xFF), 2));
 convertAllInvalidChars($invalid, $fromUnicode, 'UTF-16BE', 'BIG5', '%');
 echo "Tested UTF-16BE -> BIG5\n";
+
+// Test "long" illegal character markers
+mb_substitute_character("long");
+convertInvalidString("\x80", "BAD+80", "BIG5", "UTF-8");
+convertInvalidString("\xB0\x9F", "BAD+B09F", "BIG5", "UTF-8");
+convertInvalidString("\xA3\xED", "?+A3ED", "BIG5", "UTF-8");
```

Comment:
Might make sense to add the remaining MBFL_WCSPLANE cases to the switch, so this gets rendered as `BIG5+` rather than `?+`?

Reply:
Well, if we're going to drop that feature, it doesn't really matter.

Note that there is a difference between `BAD+XXXX` and other substitutions like `JISX+XXXX`. `BAD+XXXX` just shows raw bytes which are erroneous; `JISX+XXXX` shows not the bad bytes, but the kuten code which they *should* decode to (which doesn't actually exist in the JISX 0208 charset).

The same principle should apply to the other branches in the switch, like `BIG5+XXXX`. It should show, not raw bytes, but a non-existent 'codepoint' in the BIG5 charset, assuming that the input bytes appear like something which could actually be decoded to a codepoint in that charset.